LICENSE.md

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2018, Space Telescope Science Institute, AURA
```

Comment:
I think this is good. I've been using 'Copyright (c), AURA' in the past but I think this is probably better.

Reply:
cool. I'll go ahead and merge this then, thanks!